testutil/random.go

```diff
@@ -225,6 +270,25 @@ func RandomExecutionPayLoad() *bellatrix.ExecutionPayload {
 	}
 }
 
+func RandomExecutionPayloadHeader() *bellatrix.ExecutionPayloadHeader {
+	return &bellatrix.ExecutionPayloadHeader{
+		ParentHash:       RandomArray32(),
+		FeeRecipient:     bellatrix.ExecutionAddress{},
+		StateRoot:        RandomArray32(),
+		ReceiptsRoot:     RandomArray32(),
+		LogsBloom:        [256]byte{},
```

Comment:
you can omit "zero" values, since they are automatically assigned if not specified. So can remove all the `...{}` as well as `0`

Reply:
cool this was copied from RandomExecutionPayLoad only diff is payload has transactions whereas header has tranactionsRoot

do you want me to omit the zero values on RandomExecutionPayLoad as well?